model/clusters_mgmt/v1/cluster_configuration_mode_type.model

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Configuration mode of a cluster.
+enum ClusterConfigurationMode {
+	// Full configuration (default).
+	Full
+
+	// Only read configuration operations are supported.
```

Comment:
And can't be deleted.
Something's wrong with the indentation.

Reply:
Fixed